posthog/migrations/0410_action_steps_population.py

```diff
@@ -0,0 +1,51 @@
+# Generated by Django 4.2.11 on 2024-05-03 08:27
+
+from django.db import migrations
+from django.core.paginator import Paginator
+
+
+def copy_action_steps_to_json(apps, schema_editor):
+    from posthog.models import Action
+
+    all_actions_with_steps = Action.objects.prefetch_related("action_steps").order_by("pk").all()
```

Comment:
I think I'd still filter out objects where `steps_json` is populated so if this ends up dying and restarting it skips already processed rows

Reply:
I don't get what you mean? This is all happening transactionally so if it fails it would rollback right?

Reply:
but if the box dies after processing X pages (each bulk_update is a txn I'd guess) and the migration restarts it'd try and process the same M actions again and could get stuck never making progress.

instead if we filter it'd process some and then die and restart and pick up where it left off so might complete even though the box is OOMing periodically